### ISSUE TITLE
benkl/firefly-iii-fints-importer stuck in sync

### ISSUE THREAD

**Issue by mackshot** [bnw/firefly-iii-fints-importer#101]
Hi,

i updated to the latest docker image benkl/firefly-iii-fints-importer:latest
Digest:sha256:b48f6f1c6373c462271932401b45c1620557f042ce156d44d0a66fe9b5eeafed

Since then I got stucked after the dialog

```
Continue to next step
Continuing to next step in 2 seconds.

The connection to your bank was tested sucessfully.
```

The following POST requests will not come to an end. The log does not contain any useful information

```
[Thu Jun 29 17:59:47 2023] 192.168.5.100:40648 Accepted
[Thu Jun 29 17:59:48 2023] 192.168.5.100:40648 Closing
[Thu Jun 29 17:59:48 2023] 192.168.5.100:40664 Accepted
[Thu Jun 29 17:59:48 2023] 192.168.5.100:40664 Closing
[Thu Jun 29 17:59:50 2023] 192.168.5.100:39970 Accepted
[Thu Jun 29 17:59:50 2023] 192.168.5.100:39970 Closing
[Thu Jun 29 17:59:51 2023] 192.168.5.100:39978 Accepted
[Thu Jun 29 17:59:51 2023] 192.168.5.100:39978 Closing
[Thu Jun 29 17:59:53 2023] 192.168.5.100:39984 Accepted
```

Does anybody else have this problem after updating to the latest docker image?
How to debug it?

Thank you very much!
/Markus

**Comment by dklinger** [comment]
I am having the same issue, but one step later.
It stucks and shows a white page after hitting the Import-button.
![image](https://github.com/bnw/firefly-iii-fints-importer/assets/2249766/f29c1728-e641-429b-994b-3f59262bd6a4)

The log looks the same as yours.

**Comment by mackshot** [comment]
I have set up a new docker container using the docker image and created a new configuration as well. Since then it works for me. Nevertheless I would be happy to get an idea how to debug problems in future to be able to report or help to fix problems. @dklinger is your problem still there?

**Comment by bnw** [comment]
Yes, this is a good point. The whole import is not well guarded against an incorrect (aka old) config.json. This could be addressed e.g. with a simple json schema checking.

**Comment by aflegler** [comment]
> I am having the same issue, but one step later. It stucks and shows a white page after hitting the Import-button. ![image](https://github.com/bnw/firefly-iii-fints-importer/assets/2249766/f29c1728-e641-429b-994b-3f59262bd6a4)
> 
> The log looks the same as yours.

I had the same problem. In my case, the cause was a rule in Firefly III that was immediately deleting the imported transaction.
The following fix should resolve the issue: bnw/firefly-iii-fints-importer/#214
